load level, some xml files

sshah-1 opened the pull request:
created class to load level (still need to change its name). Also added XML files for room 1, and rooms 10-13

**SEE NEWEST COMMENT BELOW**

jacurick19 reviewed (commented):
A lot of code commented out

sshah-1 commented:
As of 12:21 03/06/21:

- finished XML files for rooms 1 and 10-17
- scaled-up dungeon elements, enemies, and a few items
- added a file to read xml/load level (named LevelLoader)
- added code in Game class to test level loader (currently commented out)
- added NPC option to LevelLoader
- added a 'floor' class for the one material in the DungeonTileSet that I couldn't find (it's the one directly to the right of the black tile)

jacurick19 reviewed (approved):
Looks good!
